testpmd/hooks/user-tests.yml

```diff
@@ -3,3 +3,21 @@
   include_role:
     name: example-cnf-validate
   when: run_migration_test|default(true)|bool
+
```

Comment:
Is it possible get the list of pods and their states? Like I would like to know if there are any pod failed because of image pull error.